packages/handlersjs-http/lib/handlers/http-cors-request.handler.spec.ts

```diff
@@ -33,6 +33,7 @@ describe('HttpCorsRequestHandler', () => {
         headers: {
           accept: 'text/plain',
           origin: 'http://test.de',
+          ['access-control-request-headers']: 'GET',
```

Comment:
not a valid value for `access-control-request-headers`: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Request-Headers

Could be a valid value for [Access-Control-Request-Method](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Request-Method)